vectorize-onedriveforbusiness-aisearch-schedule/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
-    "title": "Vectorize files on a schedule from OneDrive for Business to AI Search",
-    "description": "This workflow template runs using the schedule-based Recurrence trigger and vectorizes files from a  Microsoft OneDrive for Business folder and writes them to the AI Search vector store.",
+    "title": "Vectorize files on a schedule from Sharepoint Online to AI Search",
```

Comment:
```suggestion
    "title": "Vectorize files on a schedule from SharePoint Online to AI Search",
```

Reply:
We are using folder names as IDs - changing it is not an option at the moment. In future, we'll have ID in the manifest, then we can do that if you feel strongly about this.

Reply:
Sure, my point is that the folder name should reflect the template. "onedriveforbusiness" isn't correct for a SharePoint Online template.